lib/tasks/temporary/add_dev_navigation_links.rake

```diff
@@ -0,0 +1,57 @@
+# rubocop:disable Metrics/BlockLength
+namespace :temporary do
```

Comment:
This temporary rake task will get existing forems like DEV, thismmalife and forem.dev to the same point at which they currently are with 13 existing links on their sidebar.

Reply:
We'll remove it once we run this